Mathlib/MeasureTheory/Measure/AEMeasurable.lean

```diff
@@ -447,3 +446,23 @@ lemma MeasureTheory.NullMeasurable.aemeasurable_of_aerange {f : α → β} {t :
     lift f' to α → t using hf't
     replace hf'm : NullMeasurable f' μ := hf'm.measurable'.subtype_mk
     exact (measurable_subtype_coe.comp_aemeasurable hf'm.aemeasurable).congr hff'.symm
+
+namespace MeasureTheory
+
+lemma map_sum {ι : Type*} {m : ι → Measure α} {f : α → β}
```

Comment:
Should this lemma be protected? Also, the `map_add` version is in the `MeasureTheory.Measure` namespace.

Reply:
I haven't protected it since `map_add` isn't either. And I haven't put in the Measure namespace as there is no possible use of dot notation. But I agree that the conflict with root's `map_add` and `map_sum` are a little bit annoying. I could rename them to `map_add_measure` and `map_sum_measure`?

Reply:
Finally I have moved it to `MeasureTheory.Measure`, for coherence and to avoid the chances of clashes with the root one (it is more common to open `MeasureTheory` than `MeasureTheory.Measure`).